### PR TITLE
Add early stopping optional for auto

### PR DESCRIPTION
**Pull request by elboy3**
Comcast feedback was that they don't always want early stopping

Also removing open console url for reddit request
https://app.shortcut.com/galileo/story/6400/dq-remove-console-url

### CODE REVIEW

**Review by franz101 — COMMENTED** [review]
Looks good to me, can we test it in a colab?

**Comment by franz101** [comment]
Not passing the callback is the right approach. One separate thing is to watch out for the flag load_best_model_at_the_end

File: dataquality/core/finish.py [hunk 1]
```diff
@@ -87,7 +87,6 @@ def finish(
             "Don't close laptop or terminate shell."
         )
         wait_for_run()
-        open_console_url(res["link"])
```
Comment (setu4993) [inline dataquality/core/finish.py:90]: Can we make this dependent on a flag instead of always?

Comment (elboy3) [reply dataquality/core/finish.py:90]: Yeah we could - do you like the automatic console url? 

we agreed in slack to fully remove it https://galileoteamworkspace.slack.com/archives/C03UZ2K3D2L/p1688154343802109

Comment (setu4993) [reply dataquality/core/finish.py:90]: I do, but if that's what we've agreed to, sure, let's do it.

**Review by setu4993 — APPROVED** [review]

**Pull request merged**